### PR TITLE
feat(#1198): add tooltips for keybindings

### PR DESCRIPTION
**Pull request by danielsteinkogler**
## Basics

- [ ] The PR is rebased with current master
- [ ] I added a line to [changelog.md](/doc/changelog.md)
- [ ] Details of what I changed are in the commit messages
- [ ] References to issues, e.g. `close #X`, are in the commit messages and changelog
- [ ] The buildserver is happy

## Checklist

- [ ] I fully described what my PR does in the documentation
- [ ] I fixed all affected documentation
- [ ] I fixed the introduction tour
- [ ] I wrote migrations in a way that they are compatible with already present data
- [ ] I fixed all affected decisions
- [ ] I added automated tests or a [manual test protocol](../doc/tests/manual/protocol.md)
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I translated all strings visible to the user
- [ ] I mentioned [every code or binary](https://github.com/ElektraInitiative/PermaplanT/blob/master/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)
- [ ] I created left-over issues for things that are still to be done
- [ ] Code is conforming to [our Architecture](/doc/architecture)
- [ ] Code is conforming to [our Guidelines](/doc/guidelines)
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)
- [ ] Exceptions to any guidelines are documented

## First Time Checklist

- [ ] I have installed and I am using [pre-commit hooks](../doc/contrib/README.md#Hooks)
- [ ] I am using [Tailwind CSS Linting](https://tailwindcss.com/blog/introducing-linting-for-tailwindcss-intellisense)

## Review

- [ ] I've tested the code
- [ ] I've read through the whole code
- [ ] I've read through the whole documentation
- [ ] I've checked conformity to guidelines
- [ ] I've checked conformity to requirements
- [ ] I've checked that the requirements are tested

### CODE REVIEW

**Comment by danielsteinkogler** [comment]
jenkins build please

File: frontend/src/config/keybindings/index.ts [hunk 1]
```diff
@@ -61,6 +61,23 @@ export function getConfiguredShortcutsForAction(scope: string, action: string):
...
+/**
+ * Retrieves the tooltip string for a specific action within a specified scope, including the associated keyboard shortcut.
+ * @param scope - The scope in which to look for the associated action's shortcut.
+ * @param action - The action for which to find the associated keyboard shortcut.
+ * @param descriptionText - Optional description text to include in the tooltip.
+ * @returns The tooltip string including the keyboard shortcut if found, or just the description text if not found.
+ *         If no description text is provided, the tooltip will only include the keyboard shortcut.
+ *
+ * */
+export function getTooltipStringWithShortcut(
```
Comment (Bushuo) [inline frontend/src/config/keybindings/index.ts:73]: I would prefer a consistent nomenclature. The module is called `keybindings` and in the issue we talk about keybindings as well. Are "shortcuts" not the same as "keybindings".

Additionally, I think that keybinding is not the right word here.
We pass in an action and we get a text description of the keybinding.

So imo the function should be called `getDescriptionForKeybinding` or something.
Also imo the name of a function should not talk about the type it returns.
The functions return type has this purpose.

File: frontend/src/config/keybindings/index.ts [hunk 2]
```diff
@@ -61,6 +61,23 @@ export function getConfiguredShortcutsForAction(scope: string, action: string):
...
+  scope: string,
+  action: string,
+  descriptionText?: string,
+): string {
+  return `${descriptionText || ''} (${getConfiguredShortcutsForAction(scope, action).join(',')})`;
```
Comment (Bushuo) [inline frontend/src/config/keybindings/index.ts:78]: I have not tried if the ` ` space is visible if `descriptionText` is not passed. If so, please remove the space.
```suggestion
  return `${descriptionText || ''} (${getConfiguredShortcutsForAction(scope, action).join(',')})`.trim();
```

**Review by Bushuo — CHANGES_REQUESTED** [review]
Looks good, but please clarify the points for me 👍

**Comment by danielsteinkogler** [comment]
jenkins build please

**Comment by markus2330** [comment]
jenkins build please

**Review by markus2330 — APPROVED** [review]
Thank you, great job! :heart: 

Can you write "Esc" in the tooltip instead of "Escape" (which is what is usually on the keyboards).

I created an left-over issue #1203.

**Comment by danielsteinkogler** [comment]
@markus2330 i added translation for the keybindings

**Review by markus2330 — APPROVED** [review]
Great job!

**Pull request merged**